aria-practices.html

```diff
@@ -984,91 +1003,82 @@ <h5>Tree Popup Keyboard Interaction</h5>
+              <li><code>none</code>: When the popup is displayed, the suggested values it contains are the same regardless of the characters typed in the combobox.</li>
+              <li><code>list</code>: When the popup is triggered, it presents suggested values. If the combobox is editable,  the values complete or logically correspond to the characters typed in the combobox.</li>
               <li>
-                <code>both</code>: When the popup is triggered, it presents suggested values that complete or logically correspond to the characters typed in the textbox.
-                In addition, the portion of the selected suggestion that has not been typed by the user, known as the <q>completion string</q>, appears inline after the input cursor in the textbox.
+                <code>both</code>: When the popup is triggered, it presents suggested values that complete or logically correspond to the characters typed in the combobox.
+                In addition, the portion of the selected suggestion that has not been typed by the user, known as the <q>completion string</q>, appears inline after the input cursor in the combobox.
                 The inline completion string is visually highlighted and has a selected state.
               </li>
             </ul>
           </li>
         </ul>
-        <ol class="note">
-          <li>When referring to the roles, states, and properties documentation for the below list of patterns used for popups, keep in mind that a combobox is a single-select widget where selection always follows focus in the popup.</li>
-          <li>The roles, states, and properties for popup elements are defined in their respective design patterns:
-            <ul>
-              <li><a href="#listbox_roles_states_props">Listbox Roles, States, and Properties</a></li>
-              <li><a href="#grid_roles_states_props">Grid Roles, States, and Properties</a></li>
-              <li><a href="#tree_roles_states_props">Tree Roles, States, and Properties</a></li>
-              <li><a href="#dialog_roles_states_props">Dialog Roles, States, and Properties</a></li>
-            </ul>
-          </li>
-        </ol>
+        <div class="note">
+          <ol>
+            <li>
+              In ARIA 1.0, the combobox referenced its popup with <a href="#aria-owns" class="property-reference">aria-owns</a> instead of <a href="#aria-controls" class="property-reference">aria-controls</a>.
+              While user agents might support comboboxes with <a href="#aria-owns" class="property-reference">aria-owns</a> for backwards compatibility with legacy content, it is strongly recommended that authors use <a href="#aria-controls" class="property-reference">aria-controls</a>.
+            </li>
+            <li>When referring to the roles, states, and properties documentation for the below list of patterns used for popups, keep in mind that a combobox is a single-select widget where selection always follows focus in the popup.</li>
```

Comment:
```suggestion
            <li>When referring to the roles, states, and properties documentation for the below list of patterns used for popups, keep in mind that a combobox is a single-select widget where selection follows focus in the popup.</li>
```

Maybe delete "always", because selection does not always follow focus in tree popup.